config/samples/pulpproject_v1beta1_pulp_cr.ci.yaml

```diff
@@ -14,6 +14,11 @@ spec:
   # We have a little over 10GB free on GHA VMs/instances
   file_storage_size: "10Gi"
   container_token_secret: "container-auth"
+  pulp_settings:
+    allowed_export_paths:
+      - /tmp
+    allowed_import_paths:
+      - /tmp
```

Comment:
@ipanova I finally found out why the container tests started to fail
https://github.com/pulp/pulp_container/pull/826